src/R/Support/Impl/Help/Packages/PackageIndex.cs

```diff
@@ -70,14 +70,22 @@ public sealed class PackageIndex : IPackageIndex {
             }
         }
 
-        public Task BeforePackagesInstalledAsync(CancellationToken ct) {
+        public async Task BeforePackagesInstalledAsync(CancellationToken ct) {
+            // Package is about to be installed. Stop intellisense session
+            // so loaded packages are released and new one will not be locked.
+            // If update is pending, cancel it.
+            CancelPendingIndexUpdate();
             var timeoutCts = CancellationTokenSource.CreateLinkedTokenSource(ct);
             timeoutCts.CancelAfter(5000);
-            return _host.StopSessionAsync(timeoutCts.Token);
+            await _host.StopSessionAsync(timeoutCts.Token);
```

Comment:
Since there is no code running after `StopSessionAsync`, there is no difference in behavior with or without `await`.